### ISSUE TITLE
About the Clue Overlap and Label Overlap metrics

### ISSUE THREAD

**Issue by Smiling-Weeping-zhr** [HumanMLLM/Omni-Emotion#2]
I'd like to ask how the Clue Overlap and Label Overlap metrics used in Emotion-LLaMA were reproduced. However, based on the author’s previous GitHub issue response, it seems the metrics are difficult to reproduce due to the following reasons:

The version of the OpenAI model originally used (gpt-3.5-turbo-16k-0613) is no longer available due to updates on the OpenAI platform.

The model weights used for testing these metrics have not been released. Additionally, the label "Neutral" was removed from the training data to improve inference performance, as the EMER dataset version at the time did not include the "Neutral" class.

During the paper submission, the EMER dataset only had 100 samples. It was later expanded to 332 samples (which became the OV-MERD dataset). However, the MER2024 website no longer hosts the original EMER dataset, and the .csv file provided by the authors overlaps with OV-MERD on only 39 samples.

**Comment by Smiling-Weeping-zhr** [comment]
<img width="1249" height="648" alt="Image" src="https://github.com/user-attachments/assets/17b165e0-c664-4e86-8a08-bad3be77adb9" />
